cfme/tests/configure/test_timeprofile.py

```diff
@@ -1,85 +1,113 @@
+    soft_assert(view.time_profile_form.help_block.text == "At least one day needs to be selected")
+    soft_assert(view.time_profile_form.save_button.disabled)
+    view.time_profile_form.cancel_button.click()
 
 
 @pytest.mark.sauce
-def test_hours_required_error_validation(soft_assert):
-    tp = st.Timeprofile(
-        description='time_profile' + fauxfactory.gen_alphanumeric(),
+def test_hours_required_error_validation(appliance, soft_assert):
+    """
+    This test case performs the error validation of hours field.
+    """
+    collection = appliance.collections.time_profiles
+    collection.create(
+        description='time_profile {}'.format(fauxfactory.gen_alphanumeric()),
         scope='Current User',
         timezone="(GMT-10:00) Hawaii",
         days=True,
-        hours=False)
-    view = navigate_to(st.Timeprofile, 'All')
-    tp.create(cancel=True)
-    soft_assert(view.timeprofile_form.help_block.text == "At least one hour needs to be selected")
-    soft_assert(view.timeprofile_form.save_button.disabled)
-    view.timeprofile_form.cancel_button.click()
+        hours=False,
+        cancel=True)
+    view = appliance.browser.create_view(TimeProfileAddFormView)
+    soft_assert(view.time_profile_form.help_block.text == "At least one hour needs to be selected")
+    soft_assert(view.time_profile_form.save_button.disabled)
+    view.time_profile_form.cancel_button.click()
 
 
 @pytest.mark.sauce
-def test_timeprofile_description_required_error_validation(soft_assert):
-    tp = st.Timeprofile(
+def test_time_profile_description_required_error_validation(appliance, soft_assert):
+    """
+    This test case performs the error validation of description field.
+    """
+    collection = appliance.collections.time_profiles
+    collection.create(
         description=None,
         scope='Current User',
+        timezone="(GMT-10:00) Hawaii",
+        days=True, hours=True,
+        cancel=True)
+    view = appliance.browser.create_view(TimeProfileAddFormView)
+    soft_assert(view.time_profile_form.description.help_block == "Required")
+    soft_assert(view.time_profile_form.save_button.disabled)
+    view.time_profile_form.cancel_button.click()
+
+
+@pytest.mark.sauce
+def test_time_profile_copy(appliance):
+    """
+    This test case checks the copy operation of the time_profile.
+    """
+    collection = appliance.collections.time_profiles
+    time_profile = collection.create(
+        description='time_profile {}'.format(fauxfactory.gen_alphanumeric()),
+        scope='Current User',
+        days=True,
+        hours=True,
         timezone="(GMT-10:00) Hawaii")
-    view = navigate_to(st.Timeprofile, 'All')
-    tp.create(cancel=True)
-    soft_assert(view.timeprofile_form.description.help_block == "Required")
-    soft_assert(view.timeprofile_form.save_button.disabled)
-    view.timeprofile_form.cancel_button.click()
+    copied_time_profile = time_profile.copy()
```

Comment:
More assertion in this test
Pass a specific description for the new entity
navigate to the newly copied entity, check view.is_displayed and its details widgets for correct values